src/library_glfw.js

```diff
@@ -62,23 +62,23 @@ var LibraryGLFW = {
       this.domKeys = new Array();
       this.shouldClose = 0;
       this.title = null;
-      this.windowPosFunc = null; // GLFWwindowposfun
-      this.windowSizeFunc = null; // GLFWwindowsizefun
-      this.windowCloseFunc = null; // GLFWwindowclosefun
-      this.windowRefreshFunc = null; // GLFWwindowrefreshfun
-      this.windowFocusFunc = null; // GLFWwindowfocusfun
-      this.windowIconifyFunc = null; // GLFWwindowiconifyfun
-      this.windowMaximizeFunc = null; // GLFWwindowmaximizefun
-      this.framebufferSizeFunc = null; // GLFWframebuffersizefun
-      this.windowContentScaleFunc = null; // GLFWwindowcontentscalefun
-      this.mouseButtonFunc = null; // GLFWmousebuttonfun
-      this.cursorPosFunc = null; // GLFWcursorposfun
-      this.cursorEnterFunc = null; // GLFWcursorenterfun
-      this.scrollFunc = null; // GLFWscrollfun
-      this.dropFunc = null; // GLFWdropfun
-      this.keyFunc = null; // GLFWkeyfun
-      this.charFunc = null; // GLFWcharfun
-      this.userptr = null;
+      this.windowPosFunc = 0; // GLFWwindowposfun
```

Comment:
This change is because `null` cannot be converted to `BigInt` when we return these as pointers.